gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -2272,6 +2273,10 @@ else if (value instanceof Traversal)
      * and if the {@link Element} is a {@link VertexProperty}.  This method is the long-hand version of
      * {@link #property(Object, Object, Object...)} with the difference that the {@link VertexProperty.Cardinality}
      * can be supplied.
+     * <p/>* 
+     * If a {@link Map} is supplied and if supported by the {@link Graph} then each of the key/value pairs in the map will
+     * be added as property.  This method is the long-hand version of looping through the 
+     * {@link #property(Cardinality, Object, Object, Object...)} method for each key/value pair supplied.
```

Comment:
should this javadoc have been changed? this particular method doesn't do any looping through the `Map` does it?

Reply:
Got a bit overzealous here, it's been reverted,